recipes/pybiomart/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pybiomart" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e9eac20db921820670c646d99725b0ee279e407379e5e8c3ec7245a07425d8fe
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Perhaps add `--ignore-installed --no-deps` since we are specifying dependencies separately in requirements below?

Reply:
FYI, `{{ PYTHON }} -m pip install . -vv` actually performs `{{ PYTHON }} -m pip install --no-deps --ignore-installed .`, so no need for changes here.

Reply:
@npavlovikj Thanks for the tip! I was unaware of it.